examples/jsm/postprocessing/RenderPass.js

```diff
@@ -14,7 +14,7 @@ class RenderPass extends Pass {
 
 		this.overrideMaterial = overrideMaterial;
 
-		this.clearColor = clearColor;
+		this.clearColor = ( clearColor !== undefined ) ? clearColor : new THREE.Color(0x000000);
```

Comment:
It seems you have never tested this PR since the `THREE` inside this module leads to a runtime error. All post processing examples using this pass are now broken.